src/Microsoft.VisualStudio.SDK.Analyzers.Tests/DiagnosticVerifier.cs

```diff
@@ -165,10 +165,13 @@ protected static Project CreateProject(string[] sources, string language = Langu
                 }
             }
 
-            string nugetPackageRoot = Path.Combine(
-                Environment.GetEnvironmentVariable("USERPROFILE"),
-                ".nuget",
-                "packages");
+            string globalPackagesFolder = Environment.GetEnvironmentVariable("globalPackagesFolder");
```

Comment:
Should this be NugetGlobalPackagesFolder just to make it more explicit. I was actually looking to see if Nuget itself supports such a variable but couldn't find any.